dojo/tools/cred_scan/parser.py

```diff
@@ -0,0 +1,67 @@
+import csv
+import io
+from datetime import datetime
+
+
+from dojo.models import Finding
+
+
+class CredScanParser(object):
+
+    def get_scan_types(self):
+        return ["CredScan Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "CredScan Scan"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import CSV output of CredScan scan report."
+
+    def get_findings(self, filename, test):
+        content = filename.read()
+        if type(content) is bytes:
+            content = content.decode('utf-8-sig')
+        reader = csv.DictReader(io.StringIO(content), delimiter=',', quotechar='"')
+
+        dupes = dict()
+        for row in reader:
+            # Severity is not provided in this scanner, so marking all as info
+            severity = 'Info'
+
+            # Create the description
+            description = row.get('Description', 'Description not provided')
+            is_suppressed = row.get('IsSuppressed', None)
```

Comment:
Nope. Not sure what the field is referring to. Figured it would be useful in the description to those that use the tool and they can clarify later if it does not make sense.